velox/functions/prestosql/tests/HyperLogLogFunctionsTest.cpp

```diff
@@ -112,5 +125,217 @@ TEST_F(HyperLogLogFunctionsTest, emptyApproxSet) {
           "cardinality(empty_approx_set(0.1))", makeRowVector(ROW({}), 1)));
 }
 
+TEST_F(HyperLogLogFunctionsTest, mergeHll) {
+  // Test merging two HLL instances with different value ranges.
+  const int8_t indexBitLength = 12;
+
+  std::vector<std::string> serializedHlls;
+  {
+    SparseHll<> hll{&allocator_};
+    for (int i = 0; i < 10; i++) {
+      hll.insertHash(hashOne(i));
+    }
+    addSerializedHll(serializedHlls, indexBitLength, hll);
```

Comment:
What's the value of changing serializedHlls.emplace_back with addSerializedHll? It doesn't help reduce code duplication. Instead, try

```
std::string makeSparseHll(int start, int end, int indexBitLength) {
    SparseHll<> hll{&allocator_};
    for (int i = start; i < end; i++) {
      hll.insertHash(hashOne(i));
    }

    return serialize(indexBitLength, hll);
}

std::vector<std::string> serializedHlls;
serializedHlls.emplace_back(makeSparseHll(0, 10, indexBitLength));
serializedHlls.emplace_back(makeSparseHll(10, 20, indexBitLength));

```